libs/SalesforceSDKCore/SalesforceSDKCore/Classes/UserAccount/SFSDKAuthErrorManager.m

```diff
@@ -66,6 +68,20 @@ - (SFAuthErrorHandlerList *)populateDefaultAuthErrorHandlerList
     __weak typeof(self) weakSelf = self;
     SFAuthErrorHandlerList *authHandlerList = [[SFAuthErrorHandlerList alloc] init];
     
+    
+    // User agent disabled handler
+    self.userAgentErrorHandler =
```

Comment:
Before, this error would fall into the invalid grant category which only logs on error, so I made a separate handler to catch it but then reusing the alert from the generic error handler block
<img width="564" alt="Screenshot 2023-05-11 at 7 08 41 PM" src="https://github.com/forcedotcom/SalesforceMobileSDK-iOS/assets/840431/92ba76d6-0523-4993-b3c8-917a21e8b4dd">